site/docs/v1/tech/apis/_user-response-body.adoc

```diff
@@ -174,16 +174,51 @@ The Id of the Tenant that this User belongs to.
 [field]#user.timezone# [type]#[String]#::
 The User's preferred timezone. This can be used as a default to display instants, and it is recommended that you allow User's to change this per-session. The string will be in an https://www.iana.org/time-zones[IANA] time zone format.
 
-[field]#user.twoFactorDelivery# [type]#[String]#::
+[field]#user.twoFactor.methods``[x]``.authenticator.algorithm# [type]#[TOTPAlgorithm]#::
```

Comment:
What is that type? TOTPAlgorithm? Isn't it a string?

Reply:
Ah yes. This is the enum name.  It should be String.  Thank you for calling this out.